fluent/src/parser.js

```diff
@@ -886,7 +944,7 @@ class RuntimeParser {
 
         if ((cc >= 97 && cc <= 122) || // a-z
             (cc >= 65 && cc <= 90) || // A-Z
-             cc === 95 || cc === 47 || cc === 91) { // _/[
+             cc === 47 || cc === 91) { // /[
```

Comment:
I think this shouldn't check for `/` not `[` anymore? I'll fix it in a follow-up